pandas/tests/groupby/test_groupby.py

```diff
@@ -361,6 +363,108 @@ def f3(x):
         df2.groupby("a").apply(f3)
 
 
+def test_single_groupby_indices_output():
+    cols = [
+        "int",
+        "int_cat",
```

Comment:
Replace this with data dictionary instead, use list(data.keys()) where cols is used now.